frontend/src/lib/speech/panktiSelector.ts

```diff
@@ -0,0 +1,216 @@
+import { MicrosoftCognitiveServicesSpeechTranscriber } from './MicrosoftCognitiveServicesSpeechTranscriber'
+import { ResultCallback, Transcriber } from './Transcriber'
+import { WebSpeechApiTranscriber } from './WebSpeechApiTranscriber'
+
+export type PositionCallback = ( result: number ) => void
+
+export class PanktiSelector {
+  private static readonly transcriberNameCookieKey = 'TRANSCRIBER_NAME'
+  private static readonly msftApiKeyCookieKey = 'SPEECH_KEY'
+  private static readonly msftApiregionCookieKey = 'SPEECH_REGION'
+  private static readonly openaiApiSecretCookieKey = 'OPENAI_SECRET'

```

Comment:
This is unused

I was thinking of implementing this, but then focused on other things. forgot to remove this.